voltdbclient/url_test.go

```diff
@@ -23,28 +23,52 @@ func TestParseURL(t *testing.T) {
 	}
 
 	for _, s := range sample {
-		u, err := parseURL(s.conn)
-		if s.pass {
-			if err != nil {
-				t.Fatal(err)
-			}
-			if u.Scheme != s.scheme {
-				t.Errorf("expected %s got %s", s.scheme, u.Scheme)
-			}
-			if u.User.Username() != s.user {
-				t.Errorf("expected %s got %s", s.user, u.User)
-			}
-			p, _ := u.User.Password()
-			if p != s.passwd {
-				t.Errorf("expected %s got %s", s.passwd, p)
-			}
-			if u.Host != s.host {
-				t.Errorf("expected %s got %s", s.host, u.Host)
-			}
-		} else {
-			if err == nil {
-				t.Fatal("expected an error")
+		s := s //capture variable
+		t.Run("", func(t *testing.T) {
```

Comment:
This is not necessary, please leave the original test as it was. There is no justification here unless if I missed something.